src/hooks/useRenderCampaigns.tsx

```diff
@@ -20,6 +20,7 @@ function RecommendationComponentWrapper(props: {
 }
 
 function injectCampaigns(data: CampaignData) {
+  // @ts-expect-error not defined
   if (!window.nostojs) {
```

Comment:
Can't we use global namespace definition to overcome this and avoid ts-expect-error comment?

Reply:
That would make it global, we have the nostojs import from the nosto-js library which should take care of it, but isNostoLoaded() didn't work

Reply:
Let's come back to this later. Not a blocker.